ocaml/xapi/repository.ml

```diff
@@ -720,40 +668,41 @@ let apply_updates' ~__context ~host ~updates_info ~livepatches ~acc_rpm_updates
             )
         )
       ]
-      ) ;
-  (* Evaluate guidances *)
-  let guidances =
-    let guidances' =
-      (* EvacuateHost will be applied before applying updates *)
-      eval_guidances ~updates_info ~updates:acc_rpm_updates ~kind:Recommended
-        ~livepatches:successful_livepatches ~failed_livepatches
-      |> List.filter (fun g -> g <> Guidance.EvacuateHost)
-      |> fun l -> merge_with_unapplied_guidances ~__context ~host ~guidances:l
-    in
-    GuidanceSet.assert_valid_guidances guidances' ;
-    match failed_livepatches with
-    | [] ->
-        guidances'
-    | _ :: _ ->
-        (* There is(are) livepatch failure(s):
-         * the host should not be rebooted, and
-         * an extra pending guidance 'RebootHostOnLivePatchFailure' should be set.
-         *)
-        guidances'
-        |> List.filter (fun g -> g <> Guidance.RebootHost)
-        |> List.cons Guidance.RebootHostOnLivePatchFailure
+      )
+
+let apply_updates' ~__context ~host ~updates_info ~livepatches ~acc_rpm_updates
+    =
+  let open Guidance in
+  (* This function runs on coordinator host *)
```

Comment:
Would be better to move this line of comment up as the first line of this function as original or just change it to a function comment above the function.